packages/terra-form-select/CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Changelog
 
+* Added
+  * Added documentation updates for `Single Select`.
```

Comment:
Same as my other comment on the `terra-core-docs` CHANGELOG, this does not need to be included in the `terra-form-select` CHANGELOG.